tests/server/test_nlu_server.py

```diff
@@ -50,6 +50,20 @@ def test_classifier_server_train_get():
     r = server.requests.get("/models/classifier/predict?input=will it rain today")
     assert r.text == 'weather'
 
+def test_classifier_server_train_post():
+    data = {'hotel': ['book a hotel', 'need a nice place to stay', 'any motels near by'],
+    'weather': ['what is the weather like', 'is it hot outside'],
+    'place': ['which place is this', 'where are we', 'where are you going', 'which place is it'],
+    'name': ['What is your name', 'Who are you', 'What do i call you', 'what are you called']
+    }
+    server = NLUServer().serve(test=True)
+    clf = Classifier()
+    for y, v in data.items():
+       for x in v:
+           server.requests.post("/models/classifier/train", json={"data": {"inputs": x, "targets": y}})
```

Comment:
Can you try posting the data at once, without the loop? i.e, `server.requests.post("/models/classifier/train", json={"data":data})`?